engine-rest/engine-rest-openapi/src/main/templates/paths/history/cleanup/job/get.ftl

```diff
@@ -0,0 +1,52 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/history-cleanup/get-history-cleanup-job/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "findCleanupJob"
+      tag = "History Cleanup"
+      deprecated = true
+      summary = "Find clean up history job (GET)"
+      desc = "**Deprecated!** Use `GET /history/cleanup/jobs` instead.
+
+              Finds history cleanup job (See
+              [History cleanup](${docsUrl}/user-guide/process-engine/history/#job-progress))."
```

Comment:
The link does not exist anymore since 7.10, as far as I can see.

Reply:
Updated it to this one: https://docs.camunda.org/manual/7.9/user-guide/process-engine/history/#history-cleanup